bundles/org.openhab.binding.miio/src/main/resources/database/yeelink.light.ceiling4.json

```diff
@@ -56,7 +56,7 @@
 				"friendlyName": "Ambient Brightness",
 				"channel": "ambientBrightness",
 				"channelType": "ambientBrightness",
-				"type": "Number",
+				"type": "Dimmer",
 				"refresh": true,
 				"ChannelGroup": "actions",
 				"actions": [
```

Comment:
When you change it for a dimmer, the channel could get 0.
In that case I expect the lamp to require an off instead

```
{
						"command": "bg_set_power",
						"parameterType": "ONOFF",
						"condition": {
							"name": "BrightnessOnOff"
						}
					}
```

the action for brightness than to be suplemented with 
`"condition": {"name": "BrightnessExisting"}` similar as the regular brightness channel